pkg/kubectl/apply/list_element.go

```diff
@@ -65,3 +67,19 @@ func sliceCast(i interface{}) []interface{} {
 	}
 	return i.([]interface{})
 }
+
+// DetectConflict returns ConflictError if fields in recorded and remote of ListElement conflict
+// ConflictError Message contains details of conflict.
```

Comment:
we can remove the second comment because that is captured in definition of the ConflictError.